hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/SourceFormatAdapter.java

```diff
@@ -124,6 +261,7 @@ public InputBatch<Dataset<Row>> fetchNewDataInRowFormat(Option<String> lastCkptS
             r.getCheckpointForNextBatch(), r.getSchemaProvider());
       }
       case PROTO: {
+        //sanitizing is not done, but could be implemented if needed
```

Comment:
we can do checkArg and throw exception si sanitization is enabled for proto source.

Reply:
I put the check in SourceFormatAdapter constructor